### PR TITLE
My not so private branch

### PR DESCRIPTION
**Pull request by TempUserForNow**
All basics finished, solution more or less ready for review.

### CODE REVIEW

**Review by krisldz — COMMENTED** [review]
The code is very messed. Once you are using tabs, then spaces. It is hard to read...

File: app/javascript/grid.js [hunk 1]
```diff
@@ -1,3 +1,44 @@
+var selected = [];
```
Comment (krisldz) [inline app/javascript/grid.js]: why var instead of const / let?

File: app/javascript/grid.js [hunk 2]
```diff
@@ -1,3 +1,44 @@
...
+//nfinder (x+1,y+1,c)
+
+nfinder (x-1,y,c) //Left
+nfinder (x+1,y,c) //Right
+
+//nfinder (x-1,y-1,c)
+nfinder (x,y-1,c) //Bottom
+//nfinder (x+1,y-1,c)
+}
+}
```
Comment (krisldz) [inline app/javascript/grid.js]: It's unreadable

File: app/javascript/grid.js (hunk 1 above)
Comment (TempUserForNow) [reply app/javascript/grid.js]: agreed, there was no need to use var, changed it to let in last commit

File: app/javascript/grid.js (hunk 2 above)
Comment (TempUserForNow) [reply app/javascript/grid.js]: Cleaned up, i hope it will be more readable now

File: app/javascript/grid.js [hunk 3]
```diff
@@ -1,3 +1,35 @@
+let selected = [];
+function nfinder(x,y,c) //Finds connected elements vith use of recursion
+	{
+	let id = "block_"+x+"x"+y;
```
Comment (krisldz) [inline app/javascript/grid.js]: why **let** not **const**?

File: app/javascript/grid.js [hunk 4]
```diff
@@ -1,3 +1,35 @@
...
+	let myElem = document.getElementById(id);
```
Comment (krisldz) [inline app/javascript/grid.js]: same as above

File: app/javascript/grid.js [hunk 5]
```diff
@@ -51,7 +86,23 @@ export class BlockGrid {
     }
 
     blockClicked (e, block) {
-        console.log(e, block);
+        
+	let cut_me = e.target.id
+	let x_in = parseInt (cut_me.substring(6, 7), 10);//Get x parameter
+	let y_in = parseInt (cut_me.substring(8, 9), 10);//Get y parameter
```
Comment (krisldz) [inline app/javascript/grid.js]: why **let** not **const**?

File: app/javascript/grid.js [hunk 6]
```diff
@@ -51,7 +86,23 @@ export class BlockGrid {
...
+	let c_in = document.getElementById(e.target.id).style.backgroundColor//Get colour parameter
```
Comment (krisldz) [inline app/javascript/grid.js]: why **let** not **const**?

Comment (TempUserForNow) [reply app/javascript/grid.js]: Did some reading on ES6, will now prioritize const where possible. Fixed in last commit.

**Pull request closed without merging**